app/views/layouts/application.html.erb

```diff
@@ -9,8 +9,66 @@
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
+    <!-- Navigation -->
```

Comment:
Please review the indentation of this file.

Reply:
Done !

Reply:
Please review the indentation of this file again.